flow/Makefile

```diff
@@ -198,9 +198,15 @@ ifeq (, $(strip $(NPROC)))
 endif
 export NUM_CORES := $(NPROC)
 
-LSORACLE_PLUGIN ?= oracle
+export LSORACLE_CMD ?= $(shell command -v lsoracle)
+ifeq ($(LSORACLE_CMD),)
+  LSORACLE_CMD = $(abspath $(FLOW_HOME)/../tools/build/LSOracle/bin/lsoracle)
+endif
+
+LSORACLE_PLUGIN ?= $(abspath $(FLOW_HOME)/../tools/build/yosys/share/yosys/plugin/oracle.so)
```

Comment:
This path should be: `$(FLOW_HOME)/../tools/build/yosys/share/plugin/oracle.so`

Reply:
```
% find tools/build/yosys
tools/build/yosys
tools/build/yosys/bin
tools/build/yosys/bin/yosys
tools/build/yosys/bin/yosys-config
tools/build/yosys/bin/yosys-filterlib
tools/build/yosys/bin/yosys-smtbmc
tools/build/yosys/bin/yosys-abc
tools/build/yosys/share
tools/build/yosys/share/yosys
tools/build/yosys/share/yosys/greenpak4
tools/build/yosys/share/yosys/greenpak4/cells_latch.v
...
```
tools/build/yosys/share/yosys is correct, though it looks weird -> [/usr/local]/share/yosys -> [tools/build/yosys]/share/yosys

Reply:
it would actually make things easier for installation if we just did tools/build as the prefix, so you had tools/build/bin, tools/build/share, etc, but that's a question for another day.

Reply:
Yes, you are correct, but that is not where the module gets installed:
```
-- Install configuration: "RELEASE"
-- Installing: /OpenROAD-flow-scripts/tools/build/LSOracle/bin/lsoracle
-- Up-to-date: /OpenROAD-flow-scripts/tools/build/LSOracle/share/lsoracle/test.ini
-- Installing: /OpenROAD-flow-scripts/tools/build/yosys/share/plugin/oracle.so
```
Notice the path for the oracle.so, that might be an issue in the LSOracle repo that needs to be corrected.

Reply:
My mistake, you were talking about the makefile variable rather than the installation location, which i assumed was correct. I fixed the install location.

Reply:
looks good, I was fixated on the Makefile and didn't notice the install location. Thanks for fixing

Reply:
> it would actually make things easier for installation if we just did tools/build as the prefix, so you had tools/build/bin, tools/build/share, etc, but that's a question for another day.

I think the reasoning is that if someone wants to use different versions of the tools then they can do so. e.g. use OpenROAD that is installed with the scripts, but use a system-installed yosys.